### ISSUE TITLE
Shape support

### ISSUE THREAD

**Issue by marcprux** [skiptools/skip-ui#5]
### Discussed in https://github.com/skiptools/skip/discussions/35

<div type='discussions-op-text'>

<sup>Originally posted by **shial4** November  3, 2023</sup>
Please, Add shape support 
```
func haloView() -> some View {
        #if !SKIP
        Circle()
            .foregroundColor(contentColor ?? Color.clear)
            .frame(width: 8, height: 8)
            .offset(y: 4)
        #else
        EmptyView()
        #endif
    }
```
</div>

**Comment by aabewhite** [comment]
In progress...

**Comment by aabewhite** [comment]
Implemented in 0.3.11
